runtime/tests/runtime_integration_tests/test_authorize_and_stash.rs

```diff
@@ -1008,19 +1099,28 @@ fn test_authorize_from_load_address() {
     hasher.update(load_memory_contents);
     let fw_digest = hasher.finalize();
 
-    let image_metadata = vec![AuthManifestImageMetadata {
+    let image_metadata = AuthManifestImageMetadata {
         fw_id: u32::from_le_bytes(FW_ID_1),
         flags: flags.0,
         digest: fw_digest.into(),
         image_load_address: Addr64 {
-            lo: 0x0050_0000,
-            hi: 0x0000_0000,
+            lo: TEST_SRAM_BASE.lo,
+            hi: TEST_SRAM_BASE.hi,
         },
         ..Default::default()
-    }];
-    let auth_manifest = create_auth_manifest_with_metadata(image_metadata);
-    let mut model = set_auth_manifest_with_test_sram(Some(auth_manifest), &load_memory_contents);
-
+    };
+    let mcu_image = [0xAAu8; 256];
```

Comment:
Could we change these to not all be the same byte? (We've had problems before where got the endianness wrong when feeding to the SHA accelerator, for instance.)